models/articles.model.js

```diff
@@ -3,6 +3,9 @@ const db = require('../db/connection.js');
 exports.returnArticle = (article_id) => {
     article_id = Number(article_id)
     return db.query('SELECT * FROM articles WHERE article_id = $1',[article_id]).then(({rows}) => {
-        return rows
+        return new Promise((resolve, reject) => {
+            if (rows.length === 0) reject(new Error("Not Found"))
+            else resolve({Article:rows[0]})
+        })
     })
 }
```

Comment:
Here we would want to use the syntax Promise.reject({status: 400, msg: Bad request}) rather than creating a new Promise we want to reject the existing one to be caught by the controller.